Dockerfile-dev

```diff
@@ -1,4 +1,4 @@
-FROM node:12-alpine
+FROM node:16-alpine
```

Comment:
Why isn't USER added to this file for this service but it is added for the others?

Reply:
it is added for 2 of the services I think and it is because for those specific services I got a weird error that has something to do with npm and permissions
why did I get the error for some of the services and not for others? that's a good question
if we see this error when deploying to test env we will add the USER directive